.github/workflows/publish-HOWTOs.yml

```diff
@@ -0,0 +1,86 @@
+name: Publish HOWTOs
+on:
+  - push
+  - workflow_dispatch
+jobs:
+  build:
+    name: Build and prepare HOWTOs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: npm install
+      - name: Build packages
+        run: npm run build
+      - name: Prepare for publishing (GitHub Pages)
+        run: npm run prep
+        env:
+          PKG_HOWTOS_LOCATION: github
+          PKG_HOWTOS_PATH: ${{ github.ref_name }}
+      - name: Prepare for publishing (AWS s3 CDN)
+        run: npm run prep
+        env:
+          PKG_HOWTOS_LOCATION: aws
+          PKG_HOWTOS_PATH: ${{ github.ref_name }}
+      - name: Upload artifacts (GitHub Pages)
+        uses: actions/upload-artifact@v3
+        with:
+          if-no-files-found: error
+          name: workflow-howtos-github
+          path: public-github/
+          retention-days: 1
+      - name: Upload artifacts (AWS s3 CDN)
+        uses: actions/upload-artifact@v3
+        with:
+          if-no-files-found: error
+          name: workflow-howtos-aws
+          path: public-aws/
+          retention-days: 1
+
+  publish-gh:
+    name: Publish HOWTOs (GitHub Pages)
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: workflow-howtos-github
+          path: public-github/
+      - name: Publish
+        uses: JamesIves/github-pages-deploy-action@4.1.7
+        with:
+          branch: gh-pages # The branch the action should deploy to.
+          folder: public-github # The folder the action should deploy
+          target-folder: ${{ github.ref_name }}
```

Comment:
We extracted just the branch name from ref_name. It looks like this would use the full path? refs/heads/x

Reply:
Please see the explanation in https://github.com/built-on-openfin/workspace-starter/pull/105#discussion_r881381234